exec/cnex/global.c

```diff
@@ -166,7 +166,7 @@ TDispatch gfuncDispatch[] = {
     PDFUNC("net$socket_send",           net_socket_send),
     PDFUNC("net$socket_select",         net_socket_select),
 
-    // OS - Operating System function calls
+ // OS - Operating System function calls
```

Comment:
The only thing left in this commit is screwing up the indenting, which is fixed in the next commit anyway. Please remove this unneeded commit.